### ISSUE TITLE
Wrong fluid files used in mixtures due to environment path

### ISSUE THREAD

**Issue by mortenjs** [usnistgov/REFPROP-issues#206]
Refprop 10 uses wrong fluid files in predefined mixtures when different version of Refprop are installed.

1. Install Refprop 10 in default folder => "c:\Program Files (x86)\Refprop" is added to path
2. Install Refprop 9.1 in "c:\Tools" => "c:\Tools\Refprop" is added to path
3. In application using Refprop, load Refprop 10 dll from "c:\Program Files (x86)\Refprop" and load a predefined mixture, e.g. R513A, from "c:\Program Files (x86)\Refprop\Mixtures"
4. Refprop 10 will now use the fluid files from the Refprop 9.1 installation when calculating R513A!
5. Removing "c:\Tools\Refprop" from path solves the problem, and Refprop 10 is now using the correct fluid files.

This is leading to a lot of unexpected results on machines where several Refprop versions are installed (for comparison) or on machines using third party tools, which come with their own Refprop installation.

**Comment by ianhbell** [comment]
Indeed, I have been burned by this as well.  Two notes:

1) When you have parallel installations of REFPROP, ideally only one should be installed via the installer.  For the others it is better to just unzip the files and not set registry keys.  That way the variable RPPREFIX stays set to the right location, and only one copy of REFPROP is on the PATH
2) ALWAYS call the SETPATHdll function. This will remove the automatic fluid-finding routines.

When REFPROP is used like this, it is usually pretty safe to have parallel installations.  Otherwise, you can readily run into issues like what you are seeing.

**Comment by ianhbell** [comment]
Also, I would recommend to install none of the installations in a default location.  It avoids things working "magically".

**Comment by mortenjs** [comment]
Thanks for support!
Is there any plans to change Refprop's default fluid-file-search routine, so that this doesn't happen? 
I still think that there are 3'rd party programs coming with there own Refprop DLL where this can lead to strange results :-) Anyway, thanks again - now I know how to avoid this!

**Comment by ianhbell** [comment]
Nope, not currently.  Best to always do as I mention, and if so, even if other vendors have supplied versions of REFPROP, they should not clash with each other.  My opinion is that explicit is always better than implicit.
